#goal definition
float64 startPos
float64 endPos
uint8 traversals
string mapName
---
#result definition
bool success
---
#feedback
